client/src/components/History/CurrentHistory/HistoryPanel.vue

```diff
@@ -26,23 +29,35 @@
                     <HistoryFilters
                         class="content-operations-filters mx-3"
                         :filter-text.sync="filterText"
-                        :showAdvanced.sync="showAdvanced" />
+                        :show-advanced.sync="showAdvanced" />
                     <section v-if="!showAdvanced">
                         <HistoryDetails :history="history" v-on="$listeners" />
                         <HistoryMessages class="m-2" :history="history" />
                         <HistoryOperations
                             :history="history"
                             :filter-text="filterText"
                             :content-selection="selectedItems"
+                            :selection-size="selectionSize"
+                            :is-query-selection="isQuerySelection"
                             :show-selection="showSelection"
                             :expanded-count="expandedCount"
                             :has-matches="hasMatches(payload)"
                             @update:content-selection="selectItems"
                             @update:show-selection="setShowSelection"
-                            @reset-selection="resetSelection"
                             @hide-selection="onHideSelection"
-                            @select-all="selectItems(payload)"
+                            @reset-selection="resetSelection"
                             @collapse-all="collapseAll" />
+                        <transition name="shutterfade">
```

Comment:
I suggest to try it without the fade here. Right now it disappears in stages and there is a delay until the list of items is re-rendered.